graphql/data/marathon/deployments/client.js

```diff
@@ -0,0 +1,14 @@
+import mockResponse from './mock-data/deployments';
+
+/**
+ * Currently using a mock response.
+ *
+ * This will connect to our MarathonActions file to trigger requests
+ * and listen to action responses via the AppDispatcher.
+ */
+
+export default class ClientDeploymentsConnector {
+  get() {
+    return Promise.resolve(mockResponse);
```

Comment:
Thanks @nLight! Good catch. This file is a placeholder for the time being because I'm going to wire it up to the MarathonActions. But we'll still use clone so I'll fix it for consistency.